packages/ui/src/modals/UpdateMembershipModal/types.ts

```diff
@@ -0,0 +1,20 @@
+import { Reducer } from 'react'
+import { Account } from '../../common/types'
+
+export interface UpdateMemberForm {
+  id: string
+  name?: string
+  handle?: string
+  avatarURI?: string
+  about?: string
+  rootAccount?: Account
+  controllerAccount?: Account
+}
+
+export type Action<T> = {
+  type: keyof T
+  value?: T[keyof T]
+}
+export type FormReducer<T> = Reducer<T, Action<T>>
+
+export type Nullable<T> = { [P in keyof T]: T[P] | null }
```

Comment:
I don't like the `Nullable` name, as the name suggests to me the type is something like:
```type Nullable<T> = T | null```
But frankly I don't have a good suggestion. `WithNullableValues`?

Reply:
Sounds better, I'll use that :+1: